<launch>
  <node pkg="tf" type="static_transform_publisher" name="world_to_cam" 
    args="0 0 0.5 -1.57 0 -1.57 world camera 10" />
  <node ns="camera" pkg="image_proc" type="image_proc" name="image_proc"/>
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="mjpeg" />
    <param name="camera_frame_id" value="camera" />
    <param name="io_method" value="mmap"/>
	<param name="camera_info_url" value="file://$(find usb_cam)/cal.yml" />
	<remap from="/usb_cam/camera_info" to="camera/camera_info"/>
	<remap from="/usb_cam/image_raw" to="camera/image_raw"	/>
  </node>

  <node name="ar_pose" pkg="ar_pose" type="ar_single" respawn="false"
    output="screen">
    <param name="marker_pattern" type="string"
      value="$(find ar_pose)/data/patt.kanji"/>
    <param name="marker_width" type="double" value="79.0"/>
    <param name="marker_center_x" type="double" value="0.0"/>
    <param name="marker_center_y" type="double" value="0.0"/>
    <param name="threshold" type="int" value="100"/>
    <param name="use_history" type="bool" value="true"/>
  </node>
  <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
  	<param name="autosize" value="true" />
	<remap from="image" to="/camera/image_raw" />
  </node>
  	
</launch>
